### ISSUE TITLE
Not able to activate the "Next" button from step 5 to step 6 with Talkback

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#337]
Scenario to reproduce

Samsung Galaxy J3 Prime
Android version 7.0 and 8.0

Step 1 to 6 screens after splash screen

While Talkback is on, swiping right to move to next items from Step 1 screen, Talkback moves to each screen and continue reading. On the step 6 screen, pressing “Next” button, takes users to step 2 screen, swipe again to move to each item, Talkback reads steps 2 to 6 again. The same behaviour with each subsequent step.

There are other issues as well.

1.	When Talkback moves to each screen from Step 1, the step number is not updated at the bottom to reflect what screen is being read.

2.	Step 4 screen “How it works”, it only reads from second paragraph (Every day, it checks …) 
Step 5 screen, it reads from first paragraph missing the heading of the screen
Step 6 screen, it appears focus is on Nova Scotia and only reads from that point

3.	After step 4 screen, pressed “Next” button on the step 6 screen to go to step 5 screen.
When Talkback on step 5 screen, swipe to move to each element and Talkback moves to step 6. On the step 6 screen, double tapping on “Next” button doesn’t activate it and users are stuck on the screen. They can go back to previous screen using the “Back” button but not able to move forward using the “Next” button. 

Expected results
Content should be read in a proper order. Users should be able to activate the "Next" button.

Screenshots
![step4](https://user-images.githubusercontent.com/12256067/86174773-40c82f80-baf0-11ea-9413-28b46cc9a13d.gif)
![step5](https://user-images.githubusercontent.com/12256067/86174777-4291f300-baf0-11ea-911f-4c45f119fcb3.gif)

**Comment by qanitah** [comment]
Confirming the same behaviour on Android 10 (OnePlus 7).

**Comment by JuliannaR** [comment]
This is fixed in V15.
